PluginDirectories/1/workingon.bundle/options.json

```diff
@@ -0,0 +1,9 @@
+{
+	"options": [
+		{
+			"text": "Your WorkingOn.co Token: ",
+			"type": "text",
+			"key": "token"
+		}
```

Comment:
https://www.workingon.co/tools
that shows the token at the bottom - it's kinda obvious though, their page literally only has three useful pages - the main page where you have only an input, the activity feed, and the one with integrations from above.
can you maybe test this thing on your machine? I really can't make it work inside flashlight itself - it just always gets overwritten by search results from spotlight.